testing/git-crypt/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Laurent Arnoud <laurent@spkdev.net>
+# Maintainer: Laurent Arnoud <laurent@spkdev.net>
+pkgname=git-crypt
+pkgver=0.6.0
+pkgrel=0
+pkgdesc="Transparent file encryption in git"
+url="https://www.agwa.name/projects/git-crypt"
+arch="all"
+license="GPL-3.0-or-later"
+depends="git"
+makedepends="openssl-dev docbook-xml docbook-xsl"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/AGWA/$pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make ENABLE_MAN=yes
+}
+
+check() {
+	cd "$builddir"
+}
```

Comment:
The check function can be removed since it is not used.

Reply:
Ok done in https://github.com/alpinelinux/aports/pull/7070/commits/720d0ec991726340a1bf9cb30999aa61590a8b3c